/data/xuxiaoxu/miniconda3/envs/openscene/lib/python3.8/site-packages/MinkowskiEngine-0.5.4-py3.8-linux-x86_64.egg/MinkowskiEngine/__init__.py:36: UserWarning: The environment variable `OMP_NUM_THREADS` not set. MinkowskiEngine will automatically set `OMP_NUM_THREADS=16`. If you want to set `OMP_NUM_THREADS` manually, please export it on the command line before running a python script. e.g. `export OMP_NUM_THREADS=12; python your_program.py`. It is recommended to set it below 24.
  warnings.warn(
torch.__version__:1.9.0+cu102
torch.version.cuda:10.2
torch.backends.cudnn.version:7605
torch.backends.cudnn.enabled:True
args.world_size: 2
[2024-07-06 00:14:19,799 distill_pre.py line 141] arch_3d: MinkUNet18A
aug: True
base_lr: 0.0001
batch_size: 8
batch_size_val: 2
classes: 20
data_root: /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_3d
data_root_2d_fused_feature: /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_multiview_openseg
dist_backend: nccl
dist_url: tcp://127.0.0.1:6792
distributed: True
epochs: 100
eval_freq: 1
evaluate: True
feature_2d_extractor: openseg
feature_type: distill
ignore_label: 255
input_color: True
loop: 5
loss_type: cosine
manual_seed: 1463
mark_no_feature_to_unknown: True
model_path: None
momentum: 0.9
multiprocessing_distributed: True
ngpus_per_node: 2
power: 0.9
print_freq: 10
prompt_eng: True
rank: 0
resume: None
save_feature_as_numpy: False
save_folder: None
save_freq: 1
save_path: out_scannet_train/openseg
split: val
start_epoch: 0
test_batch_size: 1
test_gpu: [1]
test_repeats: 5
test_workers: 4
train_gpu: [2, 3]
use_shm: False
vis_gt: False
vis_input: False
vis_pred: False
voxel_size: 0.02
workers: 6
world_size: 2
[2024-07-06 00:14:19,799 distill_pre.py line 142] => creating model ...
args.data_root: /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_3d /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_multiview_openseg
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
self.data_paths-------: 1201 5
args.data_root: /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_3d /data/xuxiaoxu/code/openvocabulary/ovdet_2d/openscene/data/scannet_multiview_openseg
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
self.data_paths-------: 1201 5
Traceback (most recent call last):
  File "run/distill_pre.py", line 451, in <module>
    main()
  File "run/distill_pre.py", line 116, in main
    mp.spawn(main_worker, nprocs=args.ngpus_per_node,
  File "/data/xuxiaoxu/miniconda3/envs/openscene/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 230, in spawn
    return start_processes(fn, args, nprocs, join, daemon, start_method='spawn')
  File "/data/xuxiaoxu/miniconda3/envs/openscene/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 188, in start_processes
    while not context.join():
  File "/data/xuxiaoxu/miniconda3/envs/openscene/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 150, in join
    raise ProcessRaisedException(msg, error_index, failed_process.pid)
torch.multiprocessing.spawn.ProcessRaisedException: 

-- Process 1 terminated with the following error:
Traceback (most recent call last):
  File "/data/xuxiaoxu/miniconda3/envs/openscene/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 59, in _wrap
    fn(i, *args)
  File "/data/xuxiaoxu/code/openvocabulary/ovdet_2d/3DSS-VLG/run/distill_pre.py", line 183, in main_worker
    train_data = FusedFeatureLoader(datapath_prefix=args.data_root,
  File "/data/xuxiaoxu/code/openvocabulary/ovdet_2d/3DSS-VLG/dataset/pretrain_loader.py", line 69, in __init__
    raise Exception('0 file is loaded in the feature loader.')
Exception: 0 file is loaded in the feature loader.

